app/controllers/books_controller.rb

```diff
@@ -1,22 +1,31 @@
 class BooksController < ApplicationController
 
   def index
-    # require "pry"; binding.pry
-    if params[:sort] == 'pagesasc'
-    @books = Book.page_sort_asc
-    elsif params[:sort] == 'pagesdesc'
+    User.three_ratingest_users
+    case params[:sort]
+    when 'pagesasc'
+      @books = Book.page_sort_asc
+    when 'pagesdesc'
         @books = Book.page_sort_desc
-    elsif params[:sort] == 'publishedasc'
+    when 'publishedasc'
         @books = Book.published_sort_asc
-    elsif params[:sort] == 'publisheddesc'
+    when 'publisheddesc'
       @books = Book.published_sort_desc
-    elsif params[:sort] == 'reviewsasc'
+    when 'reviewsasc'
       @books = Book.reviews_sort_asc
-    elsif params[:sort] == 'reviewsdesc'
+    when 'reviewsdesc'
       @books = Book.reviews_sort_desc
     else
       @books = Book.all
     end
+
+    @highest_3 =Book.highest_3_rated_titles
+    @lowest_3 = Book.lowest_3_rated_titles
+    @three_ratingest_users = User.three_ratingest_users
+
```

Comment:
We plan on refactoring the index method to use some helper methods within this controller.